server.go

```diff
@@ -0,0 +1,33 @@
+package web
+
+import (
+	"net"
+	"net/http"
+
+	log "github.com/divisionone/micro-go-log"
+)
+
+// NetServer handles the initialisation of the http.Server by making sure it calls the correct startup method
+// based on the provided http.Server configuration.
+type NetServer struct{}
```

Comment:
Given this is defined and used only within the same package, is there a need for it to be exported?